crates/holochain/src/conductor/entry_def_store.rs

```diff
@@ -157,7 +159,15 @@ mod tests {
             entry_def_position: 1.into(),
         };
 
-        handle.register_dna(dna).await.unwrap();
+        let fake_agent = fake_agent_pub_key(0);
```

Comment:
We just need any agent key here to construct a cell id that we can pass to `register_dna_file()`

Reply:
https://github.com/holochain/holochain/pull/5189/commits/9fb668a02800c85d7a68f4d94c2d4baf27fe9d2c